docs/development/setup_development.rst

```diff
@@ -76,6 +78,8 @@ repository. This container has no security hardening or monitoring.
 
 To get started, you can try the following:
 
+.. note:: Both the parent directory and a sub-directory are named 'securedrop.' To get started you'll need to cd into the sub-directory. 
+
```

Comment:
This has been addressed yesterday by https://github.com/freedomofpress/securedrop/pull/3404 (thanks to your bug report ;-)